### PR TITLE
docs: Update layer config

### PR DESCRIPTION
**Pull request by MarkY-LunarG**
Update layer config doc with latest changes on enabling environment variables using the command-line.

### CODE REVIEW

**Comment by MarkY-LunarG** [comment]
Quick doc changes for the next SDK.  If someone could please review them and make sure they're understandable, that would be great!

File: LAYER_CONFIGURATION.md [hunk 1]
```diff
@@ -28,68 +28,128 @@ structure.
...
+$ export VK_LAYER_PATH=$VULKAN_SDK/share/vulkan/explicit_layers.d
+$ export DYLD_LIBRARY_PATH=$VULKAN_SDK/lib
+```
+
+### Enabling Layers
+
+Originally, the Vulkan Desktop Loader provided `VK_INSTANCE_LAYERS` to enable layers from the command-line.
+However, starting with the Vulkan Loader built against the 1.3.234 Vulkan headers, the `VK_LOADER_LAYERS_ENABLE` environment
+variable was added to allow for more easily enabling Vulkan layers.
+The newer Loaders will continue to accept the original `VK_INSTANCE_LAYERS` environment variable for some time, but it is
```
Comment (ncesario-lunarg) [inline LAYER_CONFIGURATION.md:84]: Ah yes, a good reminder to update my scripts :)

File: LAYER_CONFIGURATION.md [hunk 2]
```diff
@@ -28,68 +28,128 @@ structure.
...
+VK_INSTANCE_LAYERS=VK_LAYER_LUNARG_api_dump;VK_LAYER_KHRONOS_validation
 ```
-In this example, the api_dump layer will be called _before_ the Khronos validation layer.
 
-To activate layers located in a particular SDK installation, or layers built locally from source, specify the layer JSON manifest file directory using the `VK_LAYER_PATH` environment variable.
+Now, with `VK_LOADER_LAYERS_ENABLE`, you simply can use stars where you don't want to fill in the full name:
 
-For Linux, if Vulkan SDK 1.2.198.0 was locally installed in `/sdk` and `VULKAN_SDK=/sdk/1.2.198.0/x86_64`:
+```
+VK_LOADER_LAYERS_enable=*api_dump,*validation
```
Comment (ncesario-lunarg) [inline LAYER_CONFIGURATION.md:104]: Is `enable` purposely lower case here?

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

File: LAYER_CONFIGURATION.md (hunk 2 above)
Comment (MarkY-LunarG) [reply LAYER_CONFIGURATION.md:104]: *Facepalm*

**Comment by juan-lunarg** [comment]
Minor issue with your commit message. Make sure to run `scripts/check_code_format.py` before submitting.

**Pull request merged**